dist/bin/dotd

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env bash
+
+if [ -z "$PROG_HOME" ] ; then
```

Comment:
No need to write separate scripts for multiple entry points. See how it's done in `./bin/dotc`